com.unity.multiplayer.mlapi/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -79,6 +79,7 @@ public class NetworkSceneManager
         internal readonly Dictionary<string, uint> SceneNameToIndex = new Dictionary<string, uint>();
         internal readonly Dictionary<uint, string> SceneIndexToString = new Dictionary<uint, string>();
         internal readonly Dictionary<Guid, SceneSwitchProgress> SceneSwitchProgresses = new Dictionary<Guid, SceneSwitchProgress>();
+        internal readonly Dictionary<uint, NetworkObject> ScenePlacedObjects = new Dictionary<uint, NetworkObject>();
```

Comment:
Just to be sure, this functions the same as the previous `PendingSoftSyncObjects`? So once a scene placed object get spawned it will be removed from this list? Maybe we should call this `PendingScenePlacedObjects` or `UnspawnedScenePlacedObjects` if that is the case?